### PR TITLE
Add Android Client fix by @BaiSugar

### LINKED ISSUE #76: Not working on android
The game can't logon correctly on android. The login screen flashes and then nothing happend.
I have properly installed mitmproxy certs to /system and trusted it.
I thought it was because im running server remotely,but after i moved the server to a chroot Linux container(Arch linux) on the phone,the issue still.
The mitmproxy seems redirecting the server traffic correctly. But i just cant login.

Android 12,OSREL

### PR DESCRIPTION
**Pull request by Scirese**
Add Android Client fix by @BaiSugar
Fix #76

### CODE REVIEW

**Comment by MlgmXyysd** [comment]
More information is required. I noticed that Android client can enter normally without this patch.

**Comment by Scirese** [comment]
> More information is required. I noticed that Android client can enter normally without this patch.

If without this patch,some android client will behave the same as #76 .
Im using stable 1.0.0 to test,which not include this patch.

**Comment by Scirese** [comment]
new dev has fixed the issue. closing this pr.

**Comment by Scirese** [comment]
The issue still needs attention !!!
If without this patch,some android client will behave the same as #76 and cant login to the game !!

**Comment by xi-lu** [comment]
#150

**Review by memetrollsXD — APPROVED** [review]
Looks good, and doesn't change anything auth wise

File: src/main/java/emu/grasscutter/server/dispatch/DispatchServer.java [hunk 1]
```diff
@@ -373,7 +373,7 @@ public void start() throws Exception {
 		// Captcha
 		server.createContext( // api-account-os.hoyoverse.com
 				"/account/risky/api/check", 
-				new DispatchHttpJsonHandler("{\"retcode\":0,\"message\":\"OK\",\"data\":{\"id\":\"c8820f246a5241ab9973f71df3ddd791\",\"action\":\"\",\"geetest\":{\"challenge\":\"\",\"gt\":\"\",\"new_captcha\":0,\"success\":1}}}")
+				new DispatchHttpJsonHandler("{\"retcode\":0,\"message\":\"OK\",\"data\":{\"id\":\"none\",\"action\":\"ACTION_NONE\",\"geetest\":null}}") );
```
Comment (PinkD) [inline src/main/java/emu/grasscutter/server/dispatch/DispatchServer.java]:
```suggestion
				new DispatchHttpJsonHandler("{\"retcode\":0,\"message\":\"OK\",\"data\":{\"id\":\"none\",\"action\":\"ACTION_NONE\",\"geetest\":null}}")
```

**Review by PinkD — CHANGES_REQUESTED** [review]

**Comment by memetrollsXD** [comment]
If this still requires attention, please fix the merge conflicts. If not, I'll close the PR.

**Comment by Scirese** [comment]
> If this still requires attention, please fix the merge conflicts. If not, I'll close the PR

conflicts are solved.

**Pull request merged**